### PR TITLE
governmence: add Tencent to steering committee

### PR DESCRIPTION
**Pull request by Xunzhuo**
Hi community:

From the very early stage of EG project, I started to participate and contribute to it. And I am one of maintainers of EG teams now. I am willing to contribute to EG for a long run. Meanwhile, I am working at @Tencent in China. I am applying Tencent to be one of members of Steering Committee : ) It would be great honor to be one of members of Steering Committee.

I list some infos about Tencent below:

[Tencent](https://www.tencent.com/en-us/index.html) is a world-leading internet and technology company that develops innovative products and services to improve the quality of life of people around the world.

Founded in 1998 with its headquarters in Shenzhen, China, Tencent's guiding principle is to use technology for good. Our communication and social services connect more than one billion people around the world, helping them to keep in touch with friends and family, access transportation, pay for daily necessities, and even be entertained.

Tencent also publishes some of the world's most popular video games and other high-quality digital content, enriching interactive entertainment experiences for people around the globe.

Tencent also offers a range of services such as cloud computing, advertising, FinTech, and other enterprise services to support our clients' digital transformation and business growth.

Tencent embraces Open Source and actively particapates into communities, and also builds many popular OSS projects. GitHub org is @Tencent. 

Signed-off-by: bitliu <bitliu@tencent.com>

### CODE REVIEW

**Comment by Xunzhuo** [comment]
cc @mattklein123

**Comment by mattklein123** [comment]
LGTM. If the existing maintainers can also approve that would be great.

**Review by AliceProxy — DISMISSED** [review]

**Comment by arkodg** [comment]
I am in favor of this change
looking at the existing language around any steering committee changes https://github.com/envoyproxy/gateway/blob/main/GOVERNANCE.md#steering-committee, any changes to the steering committee would require a review from existing steering committee members 
@mattklein123 has already reviewed and approved
@zinuga @richarddli @skasisom6 @pydctw can you ptal ?

**Comment by pydctw** [comment]
lgtm from me.

File: GOVERNANCE.md
Comment (pydctw) [inline GOVERNANCE.md:34]: Change the date to the actual start date, 11/17/2022?

Comment (Xunzhuo) [reply GOVERNANCE.md]: Thanks! @pydctw : )

**Comment by richarddli** [comment]
I think there are several separate components to this change, and I believe there is merit to discussing them independently.

1. The governance doc specifies a 5 member steering committee. So this is a proposal to expand the steering committee from 5 to 6, with different end dates for members. We discussed how we wanted to avoid multiple different start dates for the SC.
2. I also note that @pydctw seems to have a different start date than the original SC members, which implies a different term. As specified in the governance, the seat (and thus the term) belongs to an org, so if she is representing VMware, I believe her seat should still start at the same time as the others.
3. Finally, the last issue, which I don't think is contentious, is adding Tencent as an SC member. I'm fine with that, assuming we can get clarity and alignment on 1 and 2.

**Comment by Xunzhuo** [comment]
Hi @richarddli, thanks for the comments.

According to contents in GOVERNANCE.md:

> During initial project bootstrap we will also explicitly reevaluate steering membership after each release; alpha; beta; GA. The existing steering committee will decide on any changes to the steering committee based on what is best for the project and community.

I think new members can be considered to be accepted. So yes, this PR also means expanding members to 6.

I think maybe we can focus on the ending time of one period (one year) of steering committees. We can have a different start time. But all members need to be reconsidered after one period ends. So we do not need to worry the differences of start time.

**Comment by skriss** [comment]
> I also note that @pydctw seems to have a different start date than the original SC members, which implies a different term. As specified in the governance, the seat (and thus the term) belongs to an org, so if she is representing VMware, I believe her seat should still start at the same time as the others.

cc @danehans (ref. https://github.com/envoyproxy/gateway/pull/720#pullrequestreview-1176264698)

**Comment by richarddli** [comment]
FYI, I'll circle with the SC members and get consensus on how to proceed!

**Comment by danehans** [comment]
Great. Thanks for taking the lead on this issue @richarddli.

**Comment by danehans** [comment]
@richarddli any updates to resolve this issue?

**Comment by richarddli** [comment]
Apologies for the delay! There is consensus from the SC on the following:

* Co-terminating everyone to 5/16/2023. So we should change the start dates in the doc to the _end_ dates, so everyone has the same end dates.
* Add Tencent, per the above, with an end date of 5/16/2023
* Update VMware to 5/16/2023 as well

**Comment by danehans** [comment]
@richarddli thanks for the update.

@Xunzhuo can you update this PR to resolve all of the feedback from https://github.com/envoyproxy/gateway/pull/735#issuecomment-1353580035?

**Comment by Xunzhuo** [comment]
Sure! Thanks.

File: GOVERNANCE.md
Comment (AliceProxy) [inline GOVERNANCE.md:29]: nit: 

> [Organizations can also be removed from the steering committee due to lack of project commitment
> on the same yearly basis. This will be determined by a majority vote of the existing committee.
> __The exception to this is the Envoy core proxy maintainer seat which is a permanent seat.__](https://github.com/envoyproxy/gateway/blob/main/GOVERNANCE.md?plain=1#L41) 

:slightly_smiling_face:

**Review by AliceProxy — CHANGES_REQUESTED** [review]

File: GOVERNANCE.md
Comment (danehans) [inline GOVERNANCE.md:24]: IMO I don't think the SC should be capped at a certain number of members. Therefore, I suggest we remove the number of members from this sentence and focus on what the SC is all about.

@richarddli @mattklein123 feel free to chime in if you disagree.

Comment (mattklein123) [reply GOVERNANCE.md:24]: That's fine with me.

Comment (danehans) [reply GOVERNANCE.md:24]: @Xunzhuo I suggest:
/The steering committee will be a 6-member body that will make directional and any strategic technical decisions on the Envoy gateway project./The steering committee is a governing body that makes directional and strategic decisions for the Envoy gateway project./

Comment (richarddli) [reply GOVERNANCE.md:24]: I would vote to keep it at 6. I know in practice it's not an issue but I also think that having an unbounded committee is not a recipe for good governance.

Comment (danehans) [reply GOVERNANCE.md:24]: @richarddli good point. I'm fine keeping it at 6.

@Xunzhuo please update as follows:

/The steering committee will be a 6-member body that will make directional and any strategic technical decisions on the Envoy gateway project./The steering committee is a 6-member governing body that makes directional and strategic decisions for the Envoy gateway project./

Comment (Xunzhuo) [reply GOVERNANCE.md:24]: Thanks, I will update it later.

File: GOVERNANCE.md
Comment (Xunzhuo) [reply GOVERNANCE.md:29]: Not sure how to resolve this one, just remove the end date of Envoy core proxy maintainers?

Comment (richarddli) [reply GOVERNANCE.md:29]: Yes, that's fine. Thanks!

Comment (Xunzhuo) [reply GOVERNANCE.md]: Done @richarddli

**Review by richarddli — APPROVED** [review]

**Review by danehans — APPROVED** [review]

**Pull request merged**
